Readme.md

```diff
@@ -53,8 +54,15 @@ Compose {
 
 ### Run Tests
 The file to run the tests is called 'run-tests.bash' and it is located in the scripts directory.
-This script will run the available test files, 'ourtests.py' and 'semantictests.py'.
+This script will run the available test files.
 All tests should be passing.  If all are not, a description of the error will be outputted.
 
     $ ./scripts/run-tests.bash
+Or a given test can be run using python.
+
+    $ python semantictest.py
+    
+    
+The CI hook is set up by running the following.
```

Comment:
Can you go a little further in depth on what the CI hook is / what it does?

Additionally, I think it should be at the top of the Run Tests section. Say it's required to run it if you want to contribute to our project.